examples/helm-template/README.md

```diff
@@ -0,0 +1,36 @@
+# Helm Template
+
+This is an example of invoking the helm template function using declarative configuration.
+
+## Function invocation
+
+The function is invoked by authoring a function configuration in local-configs/example.yaml
+with `metadata.annotations.[config.kubernetes.io/function]` set to
+`gcr.io/kpt-functions/helm-template`.
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/helm-template .
+cd helm-template; kpt cfg cat local-configs
+kpt fn run local-configs --mount type=bind,src=$(pwd)/helloworld-chart,dst=/source
```

Comment:
Can you simplify this to:

kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/helm-template .
kpt fn run helm-template/local-configs --mount type=bind,src=$(pwd)/helloworld-chart,dst=/source

you can do `kpt cfg cat` afterwards if reader is interested in learning what's in the package.

Reply:
Simplified